erizo/src/erizo/handlers/handlers/LoggerHandler.h

```diff
@@ -0,0 +1,29 @@
+#ifndef ERIZO_SRC_ERIZO_HANDLERS_HANDLERS_LOGGERHANDLER_H_
+#define ERIZO_SRC_ERIZO_HANDLERS_HANDLERS_LOGGERHANDLER_H_
+
+#include "../../pipeline/Handler.h"  // Import CustomHandler interface
+#include "../../MediaDefinitions.h"  // Imports DataPacket struct
+#include "./logger.h"  // Include logger
+
+
+namespace erizo {  // Handlers are include in erizo namespace
+class LoggerHandler : public CustomHandler {
+  DECLARE_LOGGER();  // Declares logger for debugging and logging
+ public:
+    explicit LoggerHandler(std::map <std::string, std::string> parameters);
+    ~LoggerHandler();
+    void read(Context *ctx, std::shared_ptr <DataPacket> packet) override;  // Process packet sent by client
+    void write(Context *ctx, std::shared_ptr <DataPacket> packet) override;  // Process packet sent to client
+    Positions position() override;  // Returns position to place handler.
+    void enable() override;  // Enable handler
+    void disable() override;  // Disable handler
+    std::string getName() override;  // Returns handler name
+    void notifyUpdate() override;  // Recieves update
+ private:
+    RtcpHeader* rtcp_head;
+    RtpHeader* rtp_head;
+    bool isEnabled = true;
```

Comment:
We prefer `snake_case` vs `camelCase` in the C++ part of Licode. We still see some camel cases there, but those are still inherited from the past.